### PR TITLE
Some fix on pagination

### PR DESCRIPTION
**Pull request by aseid1997**

### CODE REVIEW

File: pages/admin/property/property-list.tsx [hunk 1]
```diff
@@ -63,7 +63,7 @@ const PropertyList = ({
             </div>
           </div>
         </section>
```
Comment (hirebirhan) [inline pages/admin/property/property-list.tsx:65]: Use tailwind

File: pages/admin/property/property-list.tsx [hunk 2]
```diff
@@ -63,7 +63,7 @@ const PropertyList = ({
...
-        <section className="property-dashboard" style={{ marginTop: "26px" }}>
+        <section className="property-dashboard tw-mt-7">
```
Comment (hirebirhan) [inline pages/admin/property/property-list.tsx:66]: Use tailwind

File: src/services/property/propertyService.tsx [hunk 3]
```diff
@@ -17,7 +17,8 @@ export const getProperties = async () => {
     throw error;
   }
 };
-export const getPaginatedProperties = async (pageNumber: number, pageSize: number) => {  
+export const getPaginatedProperties = async (pageNumber: number, pageSize: number) => {
```
Comment (hirebirhan) [inline src/services/property/propertyService.tsx:20]: what if we didn't pass those params, have you defaulted the page to 1 if the param is not set

File: pages/admin/property/property-list.tsx (hunk 1 above)
Comment (aseid1997) [reply pages/admin/property/property-list.tsx:65]: chnaged

File: src/services/property/propertyService.tsx (hunk 3 above)
Comment (aseid1997) [reply src/services/property/propertyService.tsx:20]: const pageNumber = query.pageNumber ? parseInt(query.pageNumber as string) : 1;
  const pageSize = query.pageSize ? parseInt(query.pageSize as string) : 9; yeah here when I call the api if there is no value here is the default

**Review by hirebirhan — APPROVED** [review]

**Review by hirebirhan — APPROVED** [review]

**Pull request merged**